src/main/scala/it/unibo/authsim/library/user/builder/UserInformationBuilder.scala

```diff
@@ -0,0 +1,68 @@
+package it.unibo.authsim.library.user.builder
+
+import it.unibo.authsim.library.dsl.policy.defaults.PolicyDefault
+import it.unibo.authsim.library.dsl.policy.builders.StringPoliciesBuilders.UserIDPolicyBuilder
+import it.unibo.authsim.library.dsl.policy.model.Policy
+import it.unibo.authsim.library.dsl.policy.model.StringPolicies.CredentialPolicy
+import it.unibo.authsim.library.user.model.{CryptoInformation, UserInformation}
+import scala.language.postfixOps
+import it.unibo.authsim.library.user.builder.UserBuilder
+
+/**
+ * Class that represent a builder of UserInformation
+ */
+class UserInformationBuilder extends OptionalBuilder[UserInformation]:
```

Comment:
This class could/should extend `Builder[Option[UserInformation]]`, which also provide a helper method to define building methods:
```suggestion
class UserInformationBuilder extends Builder[Option[UserInformation]]:
```